main.tex

```diff
@@ -0,0 +1,423 @@
+is necessary.
+
+The local tests are run as part of the continuous integration for the
+project with each commit.  The remote tests are run as part of a
+regularly-scheduled \texttt{cron} job.  Running the remote tests infrequently
+also helps reduce the burden on the remote services.
+
+\subsection{Other utilities}
+There are several general-use utilities implemented as part of \astroquery, such
+as a bulk FITS file downloader and renamer and a download progressbar.  There
+is also a schema system implemented to allow user-side parameter validation.
+So far, at the time of writing, this tool is only implemented in the ESO and
+Vizier modules, but it could be expanded to other modules to reduce the number
+of doomed-to-fail queries sent through \astroquery.
+
+\section{Development history and status}
+\label{sec:development}
+Anyone can contribute to \astroquery.  The maintainers are committed to helping
+developers make new modules that meet the requirements of \astroquery.  This
+section describes how \astroquery has been developed, but we welcome all sorts
+of new contributions, including new modules, upgrades to existing modules, and
+minor corrections to existing tools.
+
+\astroquery is an \astropy coordinated package \citep{APE15} and is a critical
+component of the \astropy Project ecosystem \citep{Astropy-Collaboration2013a}.
+It is a standalone project and will remain independent of the \astropypkg core
+package\footnote{Many \astropy affiliated packages are developed with the
+intent of eventually including them in the core of \astropypkg.  In contrast,
+\astroquery intends to remain a separate package indefinitely largely because
+of its need to rapidly adapt to changes in the remote services; \astropypkg
+cannot make such rapid changes because users rely on its stability.}, but is
+coordinated by the \astropy Project to ensure sustainability and maintenance.
+
+\astroquery has received contributions from 77 people as of August 2018.  While the primary maintenance
+burden is shouldered by two people at any given time (the first two authors),
+most individual modules have been implemented independently by interested
+contributors.
+
+Some contributions have come as direct institutional support.  The ESA GAIA and
+ESASky modules were provided by developers working for ESA\@.  Meanwhile, the
+MAST and VO Cone Search query tools were added by developers at STSCI, with the
+latter moved over from \texttt{astropy.vo} (see Section \ref{sec:vo}).
+
+\astroquery also receives contributions from funded programs. For instance, the
+JPLHorizons module has been implemented as part of the \texttt{sbpy}
+project\footnote{\url{http://sbpy.org}} with support from NASA PDART grant
+80NSSC18K0987. Further Solar System-related services are planned to be added to
+\astroquery through this support.
+
+
+\astroquery has received support from the Google Summer of Code
+program, with two students (co-authors Madhura Parikh and Simon Liedtke)
```

Comment:
Liedtke worked on SunPy through the GSoC?

Reply:
Maybe you mean Ayush Yadav

Reply:
No, Ayush never worked on astroquery, he dropped out.  Simon did.  We picked him up the year after he worked on sunpy